spriteloader/spriteAnimated.go

```diff
@@ -64,31 +75,99 @@ type Meta struct {
 }
 
 type SpriteAnimated struct {
-	Frames map[string]interface{} `json:"frames"`
-	Meta   Meta                   `json:"meta"`
+	Frames        map[string]interface{} `json:"frames"`
+	FrameArr      []Frames
+	Meta          Meta `json:"meta"`
+	spriteSheetId SpritesheetID
 }
 
-func NewSpriteAnimated(fileName string) *SpriteAnimated {
+var spriteAnimated SpriteAnimated
+var spriteId SpriteID
+var stopPlay chan bool
+
+func NewSpriteAnimated(fileName string, lwin *render.Window) *SpriteAnimated {
```

Comment:
This function should return a new instance this time. As it stands, returning a pointer to a global variable will only support a single animated sprite